<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrearMesero</class>
 <widget class="QWidget" name="CrearMesero">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelTitulo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Crear Pedido</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIdP">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>85</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Id Pedido:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMensajeI">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>85</y>
     <width>140</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Lo ingresa el sistema.</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIdM">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>107</y>
     <width>70</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Id Mesero:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelEstado">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>133</y>
     <width>96</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Estado Pedido:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMesaheE">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>133</y>
     <width>140</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Lo ingresa el sistema.</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelProducto">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>155</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Producto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIdN">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>181</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Id Negocio:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMeIdN">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>181</y>
     <width>140</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Lo ingresa el sistema.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>203</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cantidad:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCantidad">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>203</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxIdMesero">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>107</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxProductos">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>155</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="botonCrear">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>260</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botonVolver">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Volver</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>39</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Mesa:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMesa">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
